sheepdog/blueprint/routes/views/program/project.py

```diff
@@ -410,7 +410,7 @@ def export_entities(program, project):
     project_id = "{}-{}".format(program, project)
     file_format = kwargs.get("file_format") or "tsv"
 
-    mimetype = "application/octet-stream"
+    mimetype = "application/json" if file_format.lower() == "json" else "application/octet-stream"
```

Comment:
why was this necessary?

Reply:
the unit tests are testing that when you hit the export endpoint with `format=json`, `response.json` contains the data. For other formats, `response.data` contains the data

In the new Flask code, `response.json` is None if the mimetype is not `application/json`, unless you force it to parse the data into JSON by using `response.get_json(force=True)` instead of `response.json`

I think in the older versions, doing `response.json` was forcing it by default, and throwing an error if it wasn't JSON